lib/tasks/genre.rake

```diff
@@ -0,0 +1,21 @@
+namespace :genre do
```

Comment:
Are we creating just one genre here??

Reply:
i''ll take input from user
he can run rake task again to enter another genre